src/constants.ts

```diff
@@ -96,3 +96,7 @@ export const STORAGE_KEYS = {
   LOCAL_STORAGE_COLLAB: "excalidraw-collab",
   LOCAL_STORAGE_LIBRARY: "excalidraw-library",
 };
+
+export const BACKGROUND_COLORS = {
+  TRANSPARENT: "transparent",
+};
```

Comment:
I would add it to `colors.ts` and use it there as well.. https://github.com/excalidraw/excalidraw/blob/master/src/colors.ts#L20